packages/presentation/src/__mocks__/svelte-animate.ts

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright © 2025 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// TypeScript mock for svelte/animate
+
+export type EasingFunction = (t: number) => number
+
+export interface AnimationConfig {
+  delay?: number
+  duration?: number | ((len: number) => number)
+  easing?: EasingFunction
+  css?: (t: number, u: number) => string
+  tick?: (t: number, u: number) => void
+}
+
+export interface FlipParams {
+  delay?: number
+  duration?: number | ((len: number) => number)
+  easing?: EasingFunction
+}
+
+export const flip = (params: FlipParams = {}): AnimationConfig => ({
```

Comment:
these svelte mocks look weird to me, are there any libraries that do this? If we need them, let's move to a separate package so we can reuse them in other plugins.